META-INF/com/google/android/update-binary

```diff
@@ -90,54 +90,88 @@ ui_print "******************************"
 ui_print "Powered by Magisk (@topjohnwu)"
 ui_print "******************************"
 
+# Get Installation or Uninstallation Choice from ZIP filename
+choice=$(basename "$ZIP");
+
+# Set Action depending on Choice
+case $choice in
+  *uninstall*|*Uninstall*|*UNINSTALL*) action=uninstallation;;
+  *) action=installation;;
+esac;
+
 ##########################################################################################
 # Install
 ##########################################################################################
 
-# Get the variable reqSizeM. Use your own method to determine reqSizeM if needed
-request_zip_size_check "$ZIP"
+# If Action is Installation then Install
+if [ $BOOTMODE || "$action" == "installation" ]; then
```

Comment:
This is incorrect use of both [ and $BOOTMODE, you need to use `if $BOOTMODE || [ $action == installation ]; then`

Reply:
Yea! done.
I forgot it takes the variable as strings.

Thanking you.
Best regards 👍